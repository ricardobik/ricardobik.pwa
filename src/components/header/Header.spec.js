import React from 'react';
import renderer from 'react-test-renderer';

import Header from './Header';

test('Snapshot', () => {
  const component = renderer.create(
    <Header />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
