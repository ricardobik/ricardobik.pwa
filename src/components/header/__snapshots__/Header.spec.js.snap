// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<header
  className="header"
>
  <div
    className="content"
  >
    <figure>
      <img
        alt="Logo brand"
        src="test-file-stub"
      />
    </figure>
    <h1>
      AgilePoker
    </h1>
  </div>
</header>
`;
